stores/profile.ts

```diff
@@ -0,0 +1,61 @@
+import { defineStore } from 'pinia'
+import { ProfileFormData } from '@/components/profile/create/stages'
+import {
+  CreateProfileRequest,
+  UpdateProfileRequest,
+  createProfile,
+  updateProfile,
+} from '@/services/profile'
+import { getBioWithLinks } from '@/components/profile/utils'
+import {
+  constructSocials,
+  uploadProfileImage,
+} from '@/components/profile/create/utils'
+
+interface State {
+  stage: number
+}
+
+export const useProfileStore = defineStore('profile', {
+  state: (): State => ({
+    stage: 1,
+  }),
```

Comment:
Why need to move the `stage` to a separate store? It would not be updated with the action of this store.   I would keep it a `ref` inside `components/profile/create/Modal.vue` as it's about the status of modal.

Reply:
>Why need to move the stage to a separate store? It would not be updated with the action of this store. I would keep it a ref inside components/profile/create/Modal.vue as it's about the status of modal.

I had a mental note to move it back  but I forgot , thanks!